save.go

```diff
@@ -56,19 +56,20 @@ files outside the project.
 If -t is given, test files (*_test.go files + testdata directories) are
 also saved.
 
+If -T is given, packages required by test files are not saved. -T and -t must be given at the same time.
 For more about specifying packages, see 'go help packages'.
 `,
 	Run: runSave,
 }
 
 var (
-	saveR, saveT bool
+	saveR, saveT, skipTestImports bool
 )
 
 func init() {
 	cmdSave.Flag.BoolVar(&saveR, "r", false, "rewrite import paths")
 	cmdSave.Flag.BoolVar(&saveT, "t", false, "save test files")
-
+	cmdSave.Flag.BoolVar(&skipTestImports, "T", false, "skip files that are imported by test files (it shouldn't be used with -t)")
```

Comment:
It's really "Don't process dependencies import by test files"

Also `it shouldn't be used with -t` conflicts with message above (`-T and -t must be given at the same time`).